pkg/cli/jobtemplate/describe.go

```diff
@@ -66,15 +67,28 @@ func DescribeJobTemplate(ctx context.Context) error {
 		if err != nil {
 			return err
 		}
-		for _, jobTemplate := range jobTemplates.Items {
+		for i, jobTemplate := range jobTemplates.Items {
+			// Remove managedFields
+			jobTemplate.ManagedFields = nil
```

Comment:
why remove managedfields?

Reply:
The managedfields field is not of concern to users. In addition, kubectl also hides the managedfields field by default.

Reply:
how about give a choice let user to choose weather show those fields. sometimes it is helpful to check who changed the fileds.

Reply:
I think the managedfield can be show by `kubectl get pod -oyaml` instead of describe.